client/admin/js/main.js

```diff
@@ -130,7 +153,20 @@ window.screensInit = function() {
 			if ($('#date').val() || $('#time').val()) {
 				data.dateTimeSchedule = moment(`${$('#date').val() || moment().format('YYYY-MM-DD')} ${$('#time').val() || moment().format('HH:mm')}`).valueOf();
 			}
-			api('addUrl', data);
+
+			api('addUrl', data)
+				.then(function(res){
+					const canBeViewed = res.viewable;
+
+					if(!canBeViewed){
+
+						if(troubleURLS.indexOf(data.url) === -1){
```

Comment:
We could replace this with the [includes](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/includes) method.

Example:
```
if (!troubleURLS.includes(data.url)) {
  troubleURLS.push(data.url);
}
```